astroplan/core.py

```diff
@@ -158,6 +157,52 @@ def __init__(self, location=None, timezone='UTC', name=None, latitude=None,
             raise TypeError('timezone keyword should be a string, or an '
                             'instance of datetime.tzinfo')
 
+    def astropy_to_local_time(self, astropy_time):
+        '''
+        Convert the `~astropy.time.Time` object ``astropy_time`` to a
```

Comment:
You might want to add somewhere in this docstring that it's localized via `pytz`. Although it might fit better as a high-level statement somewhere in the narrative docs (although they don't exist yet...)